code/datums/mutations/space_adaptation.dm

```diff
@@ -17,6 +17,9 @@
 	return visual_indicators[type][1]
 
 /datum/mutation/human/space_adaptation/on_acquiring(mob/living/carbon/human/owner)
+	if(HAS_TRAIT(owner, TRAIT_THERMAL_VISION))
+		to_chat(owner, "<span class='warning'>You feel your eyes twitch for a moment, before returning to normal. It seems your eyes cannot adapt while your body can survive in space!")
+		return
```

Comment:
again on the half-assedness of this pr, there already exists a variable for conflicting mutations, see dwarfism and gigantism

Reply:
Thermals are NOT a mutation, they're a trait.

Reply:
![image](https://user-images.githubusercontent.com/59849408/111044164-b685e880-843e-11eb-9bad-8b6075b0a09c.png)

Reply:
Fixed.  Please offer to help instead of wildly just spewing shit.